src/puya/__main__.py

```diff
@@ -89,6 +89,7 @@ def main() -> None:
         type=LogLevel.from_string,
         choices=list(LogLevel),
     )
+    parser.add_argument("--target-avm-version", type=int, choices=[8, 9, 10])
```

Comment:
Needs a `default` parameter

Reply:
Not sure there's much value in a test as it will need to be updated on every version release and potentially won't even have a testable example in some cases. e.g. if we move to target a new version with breaking changes there wont' be any vnext opcodes you could pick to make the check fail